Consul/Interfaces/IClusterPeeringEndpoint.cs

```diff
@@ -30,5 +31,8 @@ public interface IClusterPeeringEndpoint
 
         Task<WriteResult<ClusterPeeringTokenResponse>> GenerateToken(ClusterPeeringTokenEntry tokenEntry, WriteOptions options,
             CancellationToken ct = default);
+        Task<QueryResult<List<ClusterPeeringStatus>>> PeeringList(CancellationToken cancellationToken = default);
```

Comment:
I'm aware that in a few places we return a list, but in the majority of places we use arrays. So please use arrays for consistency.